go.mod

```diff
@@ -180,6 +180,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
+require github.com/RackSec/srslog v0.0.0-20180709174129-a4725f04ec91
```

Comment:
This library seems no longer maintained?
https://github.com/RackSec/srslog

Reply:
Yes. I use it mainly for `tcp+tls` which std syslog doesn't support. Since it is unmaintained, I shall consider implementing it myself. 😅